<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>511</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Some dude...</string>
    </property>
    <widget class="QLabel" name="firstNameLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="addressLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="dobLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Date of birth</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="firstNamelineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="addresslineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dobdateEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>401</width>
       <height>22</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2001</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
    </widget>
    <widget class="QPushButton" name="savePushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>175</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Dude Details</string>
     </property>
    </widget>
    <widget class="QLabel" name="citylabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>City</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="citylineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="statelabel">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>91</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>State</string>
     </property>
    </widget>
    <widget class="QComboBox" name="statecomboBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>85</y>
       <width>91</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ZIPcitylabel">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>92</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ZIP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ZIPlineEdit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lastNameLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lastNamelineEdit">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="showPushButton">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>175</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Dude Details</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>219</y>
      <width>501</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QTextBrowser" name="outputtextBrowser">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>501</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>firstNamelineEdit</tabstop>
  <tabstop>addresslineEdit</tabstop>
  <tabstop>citylineEdit</tabstop>
  <tabstop>statecomboBox</tabstop>
  <tabstop>ZIPlineEdit</tabstop>
  <tabstop>dobdateEdit</tabstop>
  <tabstop>savePushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
